test/e2e/connector/connector.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/opensds/opensds/contrib/connector"
+	_ "github.com/opensds/opensds/contrib/connector/iscsi"
```

Comment:
If this is not used, why importing it?  Just to test the import path is correct?

Reply:
No, this import is used for setting up `init()` method in iscsi package.